content/en/agent/logs/advanced_log_collection.md

```diff
@@ -42,26 +42,26 @@ To apply a processing rule to all logs collected by a Datadog Agent, see the [Gl
 
 ## Filter logs
 
-To send only a specific subset of logs to Datadog use the `log_processing_rules` parameter in your configuration file with the **exclude_at_match** or **include_at_match** `type`.
+To send only a specific subset of logs to Datadog, use the `processing_rules` parameter in your configuration file with the **exclude_at_match** or **include_at_match** `type`.
 
 ### Exclude at match
 
 | Parameter          | Description                                                                                        |
 |--------------------|----------------------------------------------------------------------------------------------------|
 | `exclude_at_match` | If the specified pattern is contained in the message, the log is excluded and not sent to Datadog. |
 
-For example, to **filter OUT** logs that contain a Datadog email address, use the following `log_processing_rules`:
-
 {{< tabs >}}
 {{% tab "Configuration file" %}}
 
+For example, to **filter out** logs that contain a Datadog email address, use the following `processing_rules`:
+
 ```yaml
 logs:
   - type: file
     path: /my/test/file.log
     service: cardpayment
     source: java
-    log_processing_rules:
+    processing_rules:
```

Comment:
All case in the examples in this PR can remain `log_processing_rules` since the agent uses this config key when parsing integration [configs](https://github.com/DataDog/datadog-agent/blob/688c7459c42bff1ff5e1a947d3c5ffcc8cb8c094/comp/logs/agent/config/integration_config.go#L85)
```suggestion
    log_processing_rules:
```

`processing_rules` [is only used ](https://github.com/DataDog/datadog-agent/blob/688c7459c42bff1ff5e1a947d3c5ffcc8cb8c094/pkg/config/setup/config.go#L890)for the global (datadog.yaml) config.